### PR TITLE
JavaScript Project: Awesome Books - Part #4

### PR DESCRIPTION
**Pull request by mrigorir**
> Awesome Books acts as a CRUD library where you can store books and display them on a list using objects and arrays. The data persists thanks to localStorage functions.

### For this part of the project, we have built: 

- Plain HTML page to display stored books from localStorage.

- An array of objects where the books are stored.

- An add button to add books to the collection and localStorage. 

- A Remove button to delete a book from the storage and page.

- Create a localStorage to make data persist on the browser.

- Create navbar, contact info, and footer content.

- Create 3 views witch are dynamically displayed using javascript.

### CODE REVIEW

File: js/book_list.js [hunk 1]
```diff
@@ -18,12 +29,12 @@ class BookList {
...
   static addBook(obj) {
     if (obj.title === '' || obj.author === '') {
-      message.innerHTML = "<p class='shadow-sm border text-center font-bold p-3 rounded'> Don't let the fields empty, please. </p>";
+      message.innerHTML = "<p class='shadow-sm border text-center font-bold p-3 rounded w-100'> Don't let the fields empty, please. </p>";
       BookList.fade();
     } else {
       books.push(obj);
       BookList.displayBook(obj.title, obj.author);
-      message.innerHTML = "<p class='shadow-sm border text-center font-bold p-3 rounded'> New book added to the collection </p>";
+      message.innerHTML = "<p class='shadow-sm border text-center font-bold p-3 rounded w-100'> New book added to the collection </p>";
```
Comment (Davidosky007) [inline js/book_list.js:37]: * nice job adding Flash messages 👍

File: js/book_list.js [hunk 2]
```diff
@@ -68,6 +79,32 @@ class Storage {
   }
 }
 
+class ShowViews {
+  static clear() {
+    mainContent.textContent = '';
+  }
+
```
Comment (Davidosky007) [inline js/book_list.js:86]: * The current date is not displayed below the navigation bar as stated in the project requirement, Kindly add the current date using the [Luxon Library](https://moment.github.io/luxon/#/install).
*  [See here on how to get current Date and Time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @mrigorir @GuilhermeGiachellin, 

Good job so far! 👍 
* Really love the Flash messages 💪 
* Good PR summary and
* descriptive commit messages 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!
almost git

## Required Changes ♻️

# _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @mrigorir @GuilhermeGiachellin 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**